version-catalog-extensions/src/main/kotlin/ValidateVersionCatalogTask.kt

```diff
@@ -61,6 +62,14 @@ abstract class ValidateVersionCatalogTask : DefaultTask() {
             }
     }
 
+    private fun Configuration.addPlugins(catalog: DefaultVersionCatalog) {
+        catalog.plugins
+            .forEach { plugin ->
+                val pluginNotation = "${plugin.id}:${plugin.id}.gradle.plugin:${plugin.version}"
```

Comment:
It would be great to add this link as a comment here:
https://docs.gradle.org/current/userguide/plugins.html#sec:plugin_markers